CloudSQLReplicator/google/cloud/pontem/sql/replicator/cli/main.py

```diff
@@ -622,16 +622,18 @@ def replicate_dispatcher(interactive=False, config=None):
     replicate(config)
 
 
-def allow_host(host_ip):
+def allow_host(ip):
```

Comment:
Originally it was how the argument is passed from the argument parser. I could not use -h because it is reserved for help.  In hindsight, I did not have a good reason.